core/app/models/spree/product.rb

```diff
@@ -184,13 +184,13 @@ def property(property_name)
       product_properties.find_by(property: prop).try(:value)
     end
 
-    def set_property(property_name, property_value)
+    def set_property(property_name, property_value, property_presentation = property_name)
```

Comment:
Awesome. I saw same method names in arms and remembered ruby 2.2 can spit
warnings about that.

Awesome it's not happening.

On Friday, April 17, 2015, Armin Pašalić <notifications@github.com> wrote:

> In core/app/models/spree/product.rb
> <https://github.com/spree/spree/pull/6299#discussion_r28576993>:
>
> > @@ -184,13 +184,13 @@ def property(property_name)
> >        product_properties.find_by(property: prop).try(:value)
> >      end
> >
> > -    def set_property(property_name, property_value)
> > +    def set_property(property_name, property_value, property_presentation = property_name)
>
> I could, but IMHO this way looks cleaner.
>
> AFAIK, it will not cause warnings in any ruby version (I have tried it
> with 1.8.7, 2.1.5, 2.2.1 & jRuby 9k pre1).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spree/spree/pull/6299/files#r28576993>.
>